tests/output/webpage/output_test.js

```diff
@@ -266,6 +266,7 @@ describe("Linting", function() {
     if (!isFirefox()) {
         // An exception occurs in slowparse when parsing this HTML on
         // Chrome, Safari, and phantomjs.
+        // NOTE(danielhollas): This test is failing in my local machine
```

Comment:
You could change that note "This test seems to be failing on local machines" since it's also true for my machine. I would assume it's due to a difference in the Chrome browsers but I haven't dug into it.